src/corelibs/U2Core/src/globals/GUrl.cpp

```diff
@@ -149,25 +149,26 @@ bool GUrl::operator!=(const GUrl &url) const {
 // The function converts url string to multibyte form
 // default code page is CP_THREAD_ACP
 // must use "delete" to delete returned value
-const char *GUrl::getURLStringAnsi(int codePage) const {
+QByteArray GUrl::getURLStringAnsi(int codePage) const {
 #ifdef Q_OS_WIN
     std::wstring wPath = getURLString().toStdWString();
     codePage = codePage < 0 ? CP_THREAD_ACP : codePage;
 
     DWORD buffSize = WideCharToMultiByte(codePage, 0, wPath.c_str(), -1, NULL, 0, NULL, NULL);
-    if (!buffSize)
+    if (!buffSize) {
         return nullptr;
```

Comment:
Which constructor of byte array is called here?

Reply:
?